lib/content-services/src/lib/search/components/search-filter-chips/search-widget-chip/search-widget-chip.component.html

```diff
@@ -19,7 +19,7 @@
     <mat-icon>{{ chipIcon }}</mat-icon>
 </mat-chip>
 
-<mat-menu #menu="matMenu" backdropClass="adf-search-filter-chip-menu" (closed)="onClosed()">
+<mat-menu #menu="matMenu" class="adf-search-widget-extra-width" backdropClass="adf-search-filter-chip-menu" (closed)="onClosed()">
```

Comment:
Do you think we need any condition to add for this class? I think it may affect other filters layout, if you checked them and they're fine that's fine for me as is now :D

Reply:
This class is only setting the max-width, so it would come into effect only if the the contents inside need it. The other filters layouts don't need that extra width as of now (on ACA at least) , so they are not getting affected